modules/knative-serving-advanced-config.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// <List assemblies here, each on a new line>
+// * /serverless/installing_serverless/serverless-install-config-options.adoc
+
+[id="knative-serving-advanced-config_{context}"]
+= Knative Serving supported installation configuration options
+
+This guide provides information for cluster administrators about advanced installation configuration options for Knative Serving.
+
+The YAML configuration is shown by default when you are creating Knative Serving custom resource definitions. However, you can access the form view by selecting *Edit Form* in the top right corner of the *Create Knative Serving* page.
+
+[IMPORTANT]
+====
+Do not modify any YAML contained inside the `config` field. Some of the configuration values in this field are injected by the {ServerlessOperatorName}, and modifying them will cause your deployment to become unsupported.
+====
+
+image::serving-form-view.png[Create Knative Serving form]
+
+== Controller Custom Certs
+
+If your registry uses a self-signed certificate, you must enable tag-to-digest resolution by creating a ConfigMap or Secret.
+The {ServerlessOperatorName} then automatically configures Knative Serving controller access to the registry.
+
+To enable tag-to-digest resolution, the Knative Serving controller requires access to the container registry.
+
+[IMPORTANT]
+====
+The ConfigMap or Secret must reside in the same namespace as the Knative Serving custom resource definition.
+====
+
+The following example triggers the {ServerlessOperatorName} to:
+
+. Create and mount a volume containing the certificate in the controller.
+. Set the required environment variable properly.
+
+.Example YAML
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+  namespace: knative-serving
+spec:
+  controller-custom-certs:
+    name: certs
+    type: ConfigMap
+----
+
+The following example uses a certificate in a ConfigMap named `certs` in the `knative-serving` namespace.
+
+You can also configure controller custom certs using the form view.
+The supported types are `ConfigMap` and `Secret`.
+
+If no controller custom cert  is specified, this defaults to the `config-service-ca` ConfigMap.
+
+.Example default YAML
+[source,yaml]
+----
+spec:
+  controller-custom-certs:
+    name: config-service-ca
+    type: ConfigMap
+----
+
+== High availability
+
+High availability (HA) defaults to 2 replicas per controller if no number of replicas is specified.
+
+You can set this to `1` to disable HA, or add more replicas by setting a higher integer.
+
+.Example YAML
+[source,yaml]
+----
+spec:
+  high-availability:
+    replicas: 2
+----
```

Comment:
There are more options in the form. 

![Zrzut ekranu z 2020-07-17 16-16-04](https://user-images.githubusercontent.com/540893/87795900-f7dedd80-c848-11ea-9727-e2a9a4ecb7f5.png)

Reply:
Yes, but these options shouldn't show up in our console.
AFAIK they are from upstream but for whatever reasons can't be removed even though we don't support them. I could add a note to say they're not supported I guess, but I'd rather fix the actual problem in the UI.
@markusthoemmes @jcrossley3 I think we had started talking about this before, but have you heard any news on a resolution for this? I'll try to dig out any related issues or Slack conversations.